app/src/main/java/org/jellyfin/androidtv/base/BaseActivity.java

```diff
@@ -178,4 +191,16 @@ public boolean onSearchRequested() {
         TvApp.getApplication().showSearch(this, false);
         return true;
     }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        // Search functionality requests access to RECORD_AUDIO
+        if (requestCode == TvApp.SEARCH_PERMISSION) {
```

Comment:
Why move this method to the base class? It should work, just curious since it might also work in the main class without any public method calls.

Reply:
In order for this method to be called, the class must be an instance of `Activity` which `TvApp` is not. :smile: